extensions/mqtt/processors/AbstractMQTTProcessor.cpp

```diff
@@ -85,8 +85,10 @@ void AbstractMQTTProcessor::onSchedule(const std::shared_ptr<core::ProcessContex
     logger_->log_debug("AbstractMQTTProcessor: PassWord [%s]", passWord_);
   }
   value = "";
-  if (context->getProperty(CleanSession.getName(), value) && !value.empty() &&
-      org::apache::nifi::minifi::utils::StringUtils::StringToBool(value, cleanSession_)) {
+
+  utils::optional<bool> optional_val;
```

Comment:
Minor, but a name should carry information about the purpose of the variable, not its type. Recommendation: `cleanSession_parsed` (to be consistent with the camelCase spelling of `cleanSession_`).

Reply:
The same stands for other similar occurrences.